dm/master/server_test.go

```diff
@@ -357,7 +357,7 @@ func (t *testMaster) TestCheckTask(c *check.C) {
 		Task: taskConfig,
 	})
 	c.Assert(err, check.IsNil)
-	c.Assert(resp.Result, check.IsFalse)
+	c.Assert(resp.Result, check.IsTrue)
```

Comment:
the comment in L353 should also be updated.

Reply:
ok , i will finish it at today later :smile: